recipes/libcufft/meta.yaml

```
@@ -0,0 +1,104 @@
+{% set name = "cufft" %}
```

```suggestion
{% set name = "libcufft" %}
```

It now seems to me that this naming must have been the reason for the requirements to behave differently than the other recipes.

---

Just changing for now, I suspect this means it is nicer to revert the requirements changes.